### DISCUSSION TITLE
Why this snippet is repeated 3 times?

### DISCUSSION THREAD

**Discussion by ghost** [memstechtips/UnattendedWinstall#110] [General]
https://github.com/memstechtips/UnattendedWinstall/blob/main/Standard/autounattend.xml
line 9 to line 63

**Comment by rhasselbusch** [comment]
Three entrys for three platforms (x86, arm64 and amd64).

If you take a closer look there are several sections repeated for the same reason.

**Reply by ghost** [reply]
thanks for explanation
